lightning/src/ln/msgs.rs

```diff
@@ -65,63 +65,104 @@ pub struct Init {
 /// An error message to be sent or received from a peer
 #[derive(Clone)]
 pub struct ErrorMessage {
-	pub(crate) channel_id: [u8; 32],
-	pub(crate) data: String,
+	/// The channel ID involved in the error
+	pub channel_id: [u8; 32],
+	/// Human readable error description
```

Comment:
Would be nice to include a warning against printing without sanitization of crap like control chars.

Reply:
Updated this doc, and continuing with further messages